Cargo.toml

```diff
@@ -5,8 +5,9 @@ members = [
     "lib",
     "fvm",
     "common",
-    "test_actors",
-    "test_actors/actors/*",
+    "actors",
+    "actors/actors/*",
+    "actors/test_actors/*",
```

Comment:
Reason as to wy I think such a design would not work:
1. We need to have the wasm binaries from the diverse actor crates to be available before compiling the kythera-actors crate to ensure an access to them for any other crates.
2. To be able to be compiled in wasm binaries each actor need to be compiled as a lib with crate-type = ["cdylib"] (otherwise we cannot link modules) so we can't have multiple binaries generated from one crate.